lib/src/mock_user.dart

```diff
@@ -27,7 +29,7 @@ class MockUser with EquatableMixin implements User {
     UserMetadata? metadata,
   })  : _isAnonymous = isAnonymous,
         _isEmailVerified = isEmailVerified,
-        _uid = uid,
+        _uid = uid ?? const Uuid().v4(),
```

Comment:
When using MockUser during the integration test, it's better to use a hash id instead of a fixed string because we need to interact with the server when doing the integration test.

Reply:
When using MockUser during the integration test, it's better to use a hash id instead of a fixed string because we need to interact with the server when doing the integration test I think.